e2e/suites/provider/cases/gitlab/provider.go

```diff
@@ -51,7 +53,8 @@ func newGitlabProvider(f *framework.Framework, credentials, projectID string) *g
 func newFromEnv(f *framework.Framework) *gitlabProvider {
 	credentials := os.Getenv("GITLAB_TOKEN")
 	projectID := os.Getenv("GITLAB_PROJECT_ID")
-	return newGitlabProvider(f, credentials, projectID)
+	environment := os.Getenv("GITLAB_ENVIRONMENT")
+	return newGitlabProvider(f, credentials, projectID, environment)
```

Comment:
As said above, we don't do e2e tests yet, this is dead code. However, going forward it would be definitely nice to have e2e tests for gitlab. I think we can keep this code.